docs/libertyDev.md

```diff
@@ -0,0 +1,53 @@
+## libertyDev task
+
+Start a Liberty server in dev mode. This task also invokes the `libertyCreate`, `installFeature`, and `deploy` tasks before starting the server. **Note:** This task is designed to be executed directly from the Gradle command line.  To exit dev mode, press `Control-C`, or type `q` and press Enter.
+
+Dev mode provides three key features. Code changes are detected, recompiled, and picked up by your running server. Unit and integration tests are run on demand when you press Enter in the command terminal where dev mode is running, or optionally on every code change to give you instant feedback on the status of your code. Finally, it allows you to attach a debugger to the running server at any time to step through your code.
+
+The following are dev mode supported code changes. Changes to your server such as changes to the port, server name, hostname, etc. will require restarting dev mode to be detected.  Changes other than those listed below may also require restarting dev mode to be detected.
+
+* Java source file changes and Java test file changes are detected, recompiled, and picked up by your running server.  
+* Added dependencies to your `build.gradle` are detected and added to your classpath.  Dependencies that are Liberty features will be installed via the `installFeature` task.  Any other changes to your `build.gradle` will require restarting dev mode to be detected.
+* Resource file changes are detected and copied into your `target` directory. 
+* Configuration directory and configuration file changes are detected and copied into your `target` directory.  Added features to your `server.xml` will be installed and picked up by your running server.  Adding a configuration directory or configuration file that did not previously exist while dev mode is running will require restarting dev mode to be detected.
+
+
+### Examples
+
+Start dev mode.
+```
+$ gradle libertyDev
+```
+
+Start dev mode and run tests automatically after every code change.
+```
+$ gradle libertyDev --hotTests=true
```

Comment:
Should be `gradle libertyDev --hotTests`, boolean gradle options do not take an argument and will give an error if one is provided
```
Problem configuring task :libertyDev from command line.
> Command-line option '--hotTests' does not take an argument.
```